mm2src/coins/eth.rs

```diff
@@ -524,6 +525,23 @@ impl EthCoinImpl {
     }
 }
 
+async fn get_raw_transaction_impl(coin: EthCoin, req: RawTransactionRequest) -> RawTransactionResult {
+    let hash = H256::from_str(&req.tx_hash[2..req.tx_hash.len()])
+        .map_to_mm(|e| RawTransactionError::InvalidHashError(e.to_string()))?;
+    let web3_tx = coin
+        .web3
+        .eth()
+        .transaction(TransactionId::Hash(hash))
+        .compat()
+        .await
+        .map_err(|e| RawTransactionError::InvalidHashError(e.to_string()))?;
+    let web3_tx = web3_tx.or_mm_err(|| RawTransactionError::Transport(req.tx_hash))?;
```

Comment:
It means that a transaction with selected hash doesn't exist. `Transport` doesn't look suitable for this.